CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 1.1.0 (2019-09-20)
+
+- Added demo for the fuzzy datepicker directive and added toggle switches to the datepicker component's demo [#14](https://github.com/blackbaud/skyux2-demos/pull/14)
```

Comment:
Add a period after demo. :)

Reply:
We're also missing the article before "demo." I'd go with "a" instead of "the."